File: javascript/javascript-find-sum-odd-below-number/example2.js
// https://codevscolor.com/javascript-find-sum-odd-below-number
function isOdd(n) {
  return Boolean(n % 2);
}

function findSum(no) {
  let sum = 0,
    i = 0;

  while (i < no) {
    if (isOdd(i)) {
      sum += i;
    }
    i++;
  }
  return sum;
}

console.log(findSum(100));

File: java/java-find-sum-fibonacci-series/Example2.java
// https://codevscolor.com/java-find-sum-fibonacci-series

import java.util.Scanner;

class Example2 {

    private static int findSum(int n) {
        int currentValue = 1, prevValue = 0, sum = 0, temp;

        if (n <= 0)
            return 0;

        if (n == 1)
            return 1;

        sum = 1;
        int i = 2;
        while (i < n) {
            temp = currentValue;
            currentValue = prevValue + currentValue;
            prevValue = temp;

            sum += currentValue;
            i++;
        }
        return sum;
    }

    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            System.out.println("Enter the value of n: ");
            int n = sc.nextInt();

            System.out.println("Sum = " + findSum(n));
        }
    }
}